kernel-modules/support-packages/utils.sh

```diff
@@ -33,6 +33,10 @@ use_downstream_only() {
     _check_min_version "$1" "2.9.0"
 }
 
+dont_build_support_package() {
```

Comment:
I sort of hate the name of this function; better suggestions very much welcome

Reply:
How about, `skip_version`? So when it's called:
```bash
if skip_version "$mod_ver"; then
  continue
fi
```